<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskCreationDialog</class>
 <widget class="QDialog" name="TaskCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create task</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <item>
    <widget class="QComboBox" name="task_combo_box">
     <item>
      <property name="text">
       <string>Resize image</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Change color space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use SRCNN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use FSRCNN</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="parameters_stacked_widget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="resize_page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="resize_output_size_layout" stretch="0,1,1">
         <item>
          <widget class="QLabel" name="resize_output_size_label">
           <property name="text">
            <string>Output size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="resize_x"/>
         </item>
         <item>
          <widget class="QSpinBox" name="resize_y"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="resize_keep_ratio_check_box">
         <property name="text">
          <string>Keep ratio</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="resize_interpolation_layout" stretch="0,1">
         <item>
          <widget class="QLabel" name="resize_interpolation_label">
           <property name="text">
            <string>Interpolation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="resize_interpolation_combo_box">
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>B-spline</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bilinear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blackman-Harris</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Box</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Catmull-Rom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cubic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gaussian</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lanczos3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mitchell</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Radial-lanczos3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rifman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sharp-Gaussian</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Simon</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sinc</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="resize_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="color_space_page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="color_space_layout" stretch="0,1">
         <item>
          <widget class="QLabel" name="color_space_label">
           <property name="text">
            <string>Color space</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="color_space_combo_box">
           <item>
            <property name="text">
             <string>RGB to YCbCr</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>YCbCr to RGB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGB to YCoCg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>YCoCg to RGB</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="color_space_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="srcnn_page">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="srcnn_architecture_combo_box"/>
       </item>
       <item>
        <widget class="QCheckBox" name="srcnn_split_check_box">
         <property name="text">
          <string>Split image into blocks</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="srcnn_block_size_layout" stretch="0,1">
         <item>
          <widget class="QLabel" name="srcnn_block_size_label">
           <property name="text">
            <string>Block size</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="srcnn_block_size_spin_box"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="srcnn_info_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total operations: no image&lt;/p&gt;&lt;p&gt;Approximate memory consumtion: no image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="srcnn_spaceer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="srcnn_static_info_label">
         <property name="text">
          <string>SRCNN keeps image size the same, but increases sharpness of image.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fsrcnn_page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QComboBox" name="fsrcnn_architecture_combo_box"/>
       </item>
       <item>
        <widget class="QCheckBox" name="fsrcnn_split_check_box">
         <property name="text">
          <string>Split image into blocks</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fsrcnn_block_size_layout" stretch="0,1">
         <item>
          <widget class="QLabel" name="fsrcnn_block_size_label">
           <property name="text">
            <string>Block size</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fsrcnn_block_size_spin_box"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="fsrcnn_info_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total operations: no image&lt;/p&gt;&lt;p&gt;Approximate memory consumtion: no image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="fsrcnn_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>36</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="fsrcnn_static_info_label">
         <property name="text">
          <string>SRCNN keeps image size the same, but increases sharpness of image.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="main_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>main_button_box</sender>
   <signal>accepted()</signal>
   <receiver>TaskCreationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_button_box</sender>
   <signal>rejected()</signal>
   <receiver>TaskCreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
